content/actions/creating-actions/metadata-syntax-for-github-actions.md

```diff
@@ -36,18 +36,12 @@ Action metadata files use YAML syntax. If you're new to YAML, you can read "[Lea
 
 ## `inputs`
 
-**Optional** Input parameters allow you to specify data that the action expects to use during runtime. {% data variables.product.prodname_dotcom %} stores input parameters as environment variables. Input ids with uppercase letters are converted to lowercase during runtime. We recommend using lowercase input ids.
+**Optional** Input parameters allow you to specify data that the action expects to use during runtime. {% data variables.product.prodname_dotcom %} stores input parameters as environment variables. Input values are always converted to strings. We recommend using lowercase input ids.
```

Comment:
> Input values are always converted to strings.

I would leave this out. Environment variables are strings.

Reply:
Done